pages/fixtures/combined.fixtures.ts

```diff
@@ -0,0 +1,34 @@
+import { test as base } from '@playwright/test';
+import type { Page } from '@playwright/test';
+import { LoginPage } from '../../pages/login.page';
+import { HomePage } from '../../pages/home/home.page';
+import { BillingPage } from '../../pages/billingPage';
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+type Fixtures = {
+  loggedInPage: Page;
+  homePage: HomePage;
+  billingPage: BillingPage;
+};
+
+export const test = base.extend<Fixtures>({
+  loggedInPage: async ({ page }, use) => {
+    const loginPage = new LoginPage(page);
+    await loginPage.goto();
+    await loginPage.login(process.env.TEST_EMAIL!, process.env.TEST_PASSWORD!);
+    await page.waitForURL('https://practicesoftwaretesting.com/account');
```

Comment:
```suggestion
    await page.waitForURL('/account');
```
Maybe its better not to store full url in code since we could have different envs (not for this project but...)